app/org/sagebionetworks/bridge/validators/SurveyValidator.java

```diff
@@ -205,6 +208,41 @@ private void doValidateConstraintsType(Errors errors, UIHint hint, MultiValueCon
             rejectField(errors, "uiHint",
                     "doesn't allow multiples but the '%s' UI hint gathers more than one answer", hintName);
         }
+
+        // must have an enumeration (list of question options)
+        List<SurveyQuestionOption> optionList = mcon.getEnumeration();
+        if (optionList == null || optionList.isEmpty()) {
+            rejectField(errors, "enumeration", "must have non-null, non-empty choices list");
```

Comment:
Yes, this is necessary, because we can't create a survey schema if the MultiValue questions don't have the enumerations. See https://sagebionetworks.jira.com/browse/BRIDGE-1462 to have separate validation steps for "syntactically correct" and "complete and publishable"